src/main/java/ui/components/pickers/LabelPickerState.java

```diff
@@ -0,0 +1,271 @@
+package ui.components.pickers;
+
+import backend.resource.TurboLabel;
+import util.Utility;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+public class LabelPickerState {
```

Comment:
Add header comment. Not clear what this class is for. If it represents a state, then it is just a data transfer object that doesn't have lot of logic in it?